README.md

```diff
@@ -109,8 +109,28 @@ content = ButtonBox(
 
 ### Testing
 
-The YaST team encourages to use RSpec for testing YaST code in Ruby. There is
-a plan to create a helper to allow easier testing.
+The YaST team encourages to use RSpec for testing YaST code in Ruby. To help in
+that task, this gem includes some RSpec extensions. In order to use them all,
+the following line must be added to the tests.
+
+```ruby
+require 'yast/rspec'
+```
+
+For example, the following code makes use of the #path helper provided by
+{Yast::RSpec::SCR} (the only extension currently provided).
```

Comment:
I think when we add another helps we forgot to remove this note, so I think we should add it there as it do not provide anything usefull. Maybe better to point to rubydoc page for documentaiton for all rspec helpers.